### PR TITLE
chore(docs): refactor footer to use tokens

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

**Review by realdreamer — APPROVED** [review]

File: packages/apps/docs/src/components/Layout/components/Footer/styles.css.ts [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
+    backgroundColor: 'layer-1.inverse.default',
   }),
   {
-    marginBlockStart: vars.sizes.$40,
+    selectors: {
+      [`${darkThemeClass} &`]: {
+        backgroundColor:
+          tokens.kda.foundation.color.background['layer-1'].default,
+      },
+    },
```
Comment (eileenmguo) [inline packages/apps/docs/src/components/Layout/components/Footer/styles.css.ts:16]: you should not need this becuase it should be applied correctly with the atoms utility class!

File: packages/apps/docs/src/components/Layout/components/Footer/styles.css.ts [hunk 2]
```diff
@@ -32,37 +39,44 @@ export const footerClass = style([
...
+    color: 'text.subtlest.inverse.default',
     textAlign: 'center',
   }),
   {
-    paddingInline: vars.sizes.$3,
+    selectors: {
+      [`${darkThemeClass} &`]: {
+        color: tokens.kda.foundation.color.text.subtlest.default,
+      },
+    },
```
Comment (eileenmguo) [inline packages/apps/docs/src/components/Layout/components/Footer/styles.css.ts:59]: same here

File: packages/apps/docs/src/components/Layout/components/Footer/styles.css.ts [hunk 3]
```diff
@@ -32,37 +39,44 @@ export const footerClass = style([
...
+    color: 'text.subtlest.inverse.default',
     textAlign: 'center',
   }),
   {
-    paddingInline: vars.sizes.$3,
+    paddingInline: tokens.kda.foundation.size.n3,
     selectors: {
+      [`${darkThemeClass} &`]: {
+        color: tokens.kda.foundation.color.text.subtlest.default,
+      },
```
Comment (eileenmguo) [inline packages/apps/docs/src/components/Layout/components/Footer/styles.css.ts:76]: is it intentional that these do not use the same token in dark and light mode?

**Review by eileenmguo — COMMENTED** [review]
looking great!

**Review by eileenmguo — APPROVED** [review]

**Pull request merged**